App/Logic/ExposureDetectionLogic.swift

```diff
@@ -238,6 +324,30 @@ extension Logic.ExposureDetection {
       }
     }
   }
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/ExposureDetectionLogic.swift;
  Line: 327;
-->

- :warning: swiftlint (0.39.2): Lines should not have trailing whitespace. (trailing_whitespace)